### ISSUE TITLE
[Suggestion] Path to Test Folder could be parametrized and not hardcoded

### ISSUE THREAD

**Issue by portal7** [vvenegasv/exportable#3]
After cloning the repository and running the unit tests, I see that all the tests fail.

Debugging, I see that there is a test destination route that is harked.
My suggestion would be, would not it be advisable to return this parameterizable path?

**Comment by vvenegasv** [comment]
Hi, thanks for your feedback. I'm thinking add this configuration on app settings, in unit testing project. What do you think?

**Comment by portal7** [comment]
Its a good Idea!  ( BTW:  Do you plan upgrade NPOI's version?) to 2.4 ?

**Comment by vvenegasv** [comment]
Yes, I will upgrade to the latest dependencies, unless it broke the package
